hypha/apply/funds/templates/funds/modals/update_author_form.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n static %}
+{% modal_title %}{% trans "Update Author" %}{% endmodal_title %}
+
+<form
+    class="p-4 form"
+    id="author_form"
+    hx-target="this"
+    hx-swap="outerHTML"
+    method="POST"
+    hx-post="{% url 'funds:submissions:change_author' pk=object.pk %}"
+>
+    <div>
+        <dl class="mb-4">
+            <dt class="font-semibold">{% trans "Current Author" %}</dt>
+            <dd class="truncate">{{ object.user }} &lt;{{object.user.email}}&gt;</dd>
+        </dl>
+    </div>
+
+    {% url 'funds:submissions:change_author' pk=object.pk as author_update_url %}
+    {% include 'funds/includes/dialog_form_base.html' with form=form value=value %}
+</form>
```

Comment:
The "lead" modal is much simpler, no form in form. Here there seems to be two htmx calls, is that needed?

I see that the partner modal looks like above, I suspect that is overcomplicated as well.